dashboard/modules/snapshot/tests/test_job_submission.py

```diff
@@ -9,8 +9,10 @@
 import pytest
 import requests
 
-from ray._private.test_utils import (
+import ray
+from ray._private.test_utils import (  # noqa: F401
```

Comment:
why is the `noqa` here?

Reply:
This was originally needed because fixtures don't follow the import lint rules, but I updated to move the fixture to a `conftest.py` file so this lint ignore can be removed